django_lti_tool_provider/models.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+from django.db import models
+from django.contrib.auth.models import User
+from jsonfield import JSONField
+
+from ims_lti_py.tool_provider import DjangoToolProvider
+from django.conf import settings
+
+
+_logger = logging.getLogger(__name__)
+
+
+class LtiUserData(models.Model):
+    user = models.OneToOneField(User)
+    edx_lti_parameters = JSONField(default={})
```

Comment:
@smarnach it's not user model - it's a dedicated model referencing user model. It might make sense to include course ID as well, but unfortunately there's no notion of course in LTI spec, so it's only possible with custom fields; and we've got a mechanism for that already.